build/git.go

```diff
@@ -52,3 +52,12 @@ func gitAdd(root, file string) error {
 	}
 	return nil
 }
+
+func gitRm(root, file string) error {
```

Comment:
Same suggestion as in https://github.com/bblfsh/sdk/pull/395#discussion_r274486254 can be applied in here - using variadic function argument would allow to avoid [the loop](https://github.com/bblfsh/sdk/pull/395/commits/3f68b6466d97bb8bf6b7034d5549805354019186#diff-e8578c8c0c993fd45844c786dd815617R325).